### ISSUE TITLE
Template for WebStorm doesn't ignore modules.xml and [projectname].iml

### ISSUE THREAD

**Issue by outofrange** [joeblau/gitignore.io#199]
Just used gitignore.io (again, love it!) for a WebStorm project, but two files aren't ignored in my project:

- modules.xml
- tetra.iml (or whatever the project name is)

You have already patched it with https://github.com/joeblau/gitignore.io/commit/b13db933f3ab9a08ce9b9ea830eca05c8b0326e9 for IntelliJ, but not for the other products.

I would create a pull request, but I'm not sure about the best approach.
As far as I can tell, you are just using symbolic links to [github:gitignore/Global/JetBrains.gitignore](https://github.com/github/gitignore/blob/cc542de017c606138a87ee4880e5f06b3a306def/Global/JetBrains.gitignore) for every single product/term (IntelliJ; WebStorm, ...), but I guess there is no way to just create a patch for Jetbrains.gitignore, and all linked gitignores would include it?

**Comment by outofrange** [comment]
Oh. I could create a PR for GitHub's JetBrains.gitignore as well...

Is there any reason you've created a patch for IntelliJ, instead of contributing to their repository?

**Comment by joeblau** [comment]
Awesome, glad you're enjoying using the library.

> Is there any reason you've created a patch for IntelliJ, instead of contributing to their repository?

Takes too long and may not be accepted.  When I started this project, there were over 100 outstanding pull requests for updates to templates.  I even write a blog post[1] on it once one of the GitHub engineers started merging the PRs.  I just gave up on submitting PR's and make changes on this library since i merge within a day or two.  The master Repository is also pretty conservative about merging changes without valid documentation.  I found it a lot easier to just amend what they have until they get around to fixing their stuff.

Frankly, the easily thing to do would be to symlink the IntelliJ patch, then submit a PR to GitHub.

[1] - https://blog.joeblau.com/gitignore-io-update
